docs/changes.rst

```diff
@@ -23,6 +23,14 @@ information to the contrary.
 
 
 ------------------
+3.8.4 - 2017-05-14
+------------------
+
+Instead of unconditionally importing ``sqlite3``, Hypothesis now imports it
+when a SQLite database is first used.  This was the major blocker for BSD
+support with default settings.
```

Comment:
I'd probably just say "this allows Hypothesis to work on implementations compiled without sqlite support" or similar. It's not really a *major* blocker because the fix is just "install sqlite support".